shelley/design-spec/delegation_design_spec.tex

```diff
@@ -3498,7 +3603,7 @@ \subsubsection{\texorpdfstring{\(\rho\)}{\textbackslash{}rho}}
     \approx
     0.0019.
 \] This would correspond to reducing the remaining amount of available
-ADA by \({1.0019}^{73}-1\approx 0.144=14.4\%\) per year (which sounds
+ada by \({1.0019}^{73}-1\approx 0.144=14.4\%\) per year (which sounds
 awfully high\ldots).
 
 \subsubsection{\texorpdfstring{\(\tau\)}{\textbackslash{}tau}}
```

Comment:
The section about parameter \gamma should be removed, because \gamma is no longer used.

Reply:
Good catch, thanks!

Reply:
Very welcome!